examples/practice-typescript/src/components/SideBarMenu/SideBarMenu.tsx

```diff
@@ -0,0 +1,25 @@
+import SideBarMenuItem from 'components/SideBarMenuItem/SideBarMenuItem';
+import './sideBarMenu.scss';
+
+interface SideBarMenuProps {
+  active: string;
+}
+
+const SideBarMenu = ({ active }: SideBarMenuProps) => {
+  return (
+    <ul className="aside__list">
+      <SideBarMenuItem
+        text="User management"
+        router="javascript:void(0)"
+        status={active === 'User management' ? 'aside__active' : ''}
```

Comment:
we should compare current path, not the label

Reply:
Fixed